README.md

```diff
@@ -67,7 +67,7 @@ Basic Usage
 
 ## [Shallow Rendering](/docs/api/shallow.md)
 
-```jsx
+```javascript
```

Comment:
It seems like Github doesn't support JSX syntax highlighting very well. If you look at https://github.com/airbnb/enzyme/blob/master/src/ReactWrapperComponent.jsx you'll see there's very little highlighting going on